javascript/webpacker_react-npm-module/src/index.js

```diff
@@ -1,10 +1,42 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 
+const CLASS_ATTRIBUTE_NAME = 'data-react-class';
+const PROPS_ATTRIBUTE_NAME = 'data-react-props';
+
 var WebpackerReact = {
   eventsRegistered: false,
   registeredComponents : {},
+  wrapForHMR: null,
+
+  render: function(node, component) {
+    var propsJson = node.getAttribute(PROPS_ATTRIBUTE_NAME);
+    var props = propsJson && JSON.parse(propsJson);
+
+    let reactElement = React.createElement(component, props);
+    if (this.warpForHMR) {
```

Comment:
Good catch. My understanding is that it's main function is error handling
https://github.com/gaearon/react-hot-loader/blob/next/src/AppContainer.dev.js#L43